src/main/java/Level7.java

```diff
@@ -0,0 +1,136 @@
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.util.*;
+import java.io.FileWriter;
+public class Level7 {
+    public static void main(String[] args) throws IOException {
+        String input;
+        ArrayList<Task> task = new ArrayList<Task>();
+        Scanner in = new Scanner(System.in);
+        FileInputStream fis=new FileInputStream("D:/ip/src/main/data/duke.txt");
```

Comment:
You may want to consider defining filename as a constant, as it does not change at runtime.